portalnet/src/overlay_service.rs

```diff
@@ -1231,6 +1231,7 @@ where
         let node_addr = self.discovery.cached_node_addr(source).ok_or_else(|| {
             OverlayRequestError::AcceptError("unable to find ENR for NodeId".to_string())
         })?;
+        let enr_str = node_addr.enr.to_base64();
```

Comment:
If you're not going with the `if enabled` check here, then why even create this variable? Looks like it's only used in the `tracing!` log, so let's just move it there? 
```rust
        trace!(
            protocol = %self.protocol,
            request.source = %source,
            cid.send = cid.send,
            cid.recv = cid.recv,
            enr = node_addr.enr.to_base64(),
            request.content_keys = ?content_keys_string,
            "Content keys handled by offer",
        );
```

That being said, I don't understand why you removed the `if` check? I don't want to strongarm you here if you feel strongly about leaving the check in. I appreciate you running the profiling. It's very helpful to have data when making these decisions. Could we do something like 

```rust 
    if enabled!(Level::TRACE) {
        trace!(
            protocol = %self.protocol,
            request.source = %source,
            cid.send = cid.send,
            cid.recv = cid.recv,
            enr = node_addr.enr.to_base64(),
            request.content_keys = ?content_keys_string,
            "Content keys handled by offer",
        );
    }
 ```

Though my interpretation of what the maintainer said is that this is essentially what is happening, so there's no need for the `if enabled!(Level::TRACE) {}` check